tools/generators/target_build_settings/src/Generator/ProcessSwiftArgs.swift

```diff
@@ -114,9 +158,8 @@ extension Generator.ProcessSwiftArgs {
         guard let tool = try await iterator.next(),
               tool != Generator.argsSeparator
         else {
-            return (false, [], [], [])
+            return (false, [], [], [], [])
         }
-        _ = try await iterator.next()
```

Comment:
Why was this removed?

Reply:
oops, removed by mistake. let me put this back